[ADD] Payroll - 19.0 Australian localization (#15280)
odoo/documentation: content/applications/hr/payroll/payroll_localizations/australia.rst

The status changes to :guilabel:`Employer Details`, and displays the contact information for both
the company and the selected payroll responsible. Ensure the :guilabel:`Employer Details` and
:guilabel:`Employee Contact` are correct on the form.

If any information needs to be updated, exit the form and update the company or employee information
in the **Settings** app or **Employees** app, then return to the **Payroll** settings page. Click
the :guilabel:`Start Payroll Onboarding` button, and the pop-up window loads to the
:guilabel:`Employer Details` again.

.. image:: australia/au-employer-details.png
   :alt: The employer and responsible page of the onboarding pop-up window.

When the displayed information is correct, click :guilabel:`Next Step` and the status changes to
:guilabel:`Bank Details`. Using the drop-down menu, select the :guilabel:`Bank Journal` for payroll
using the drop-down menu. After a journal is selected, the :guilabel:`Bank Name`, :guilabel:`Bank
Name`, :guilabel:`Account Name`, :guilabel:`Bank Account Number`, and :guilabel:`BSB` fields are
populated with the selected bank journal's details, and **cannot** be modified. If any changes are

> needed, exit the form and update the information for the journal in the **Accounting** app.

Reviewer: When the displayed information is correct, click **Next Step** and the status changes to **Bank Details**. Using the drop-down menu, select the **Bank Journal** for payroll. After a journal is selected, the **Bank Name**, **Account Name**, **Bank Account Number,** and **BSB** fields are populated with the selected bank journal’s details, and **cannot** be modified. If any changes are needed, exit the form and update the information for the journal in the **Accounting** app.

deductions, etc. in the sections below.

Contract overview section
~~~~~~~~~~~~~~~~~~~~~~~~~

This section holds information that drives salary calculations.

Keep the :guilabel:`Salary Structure Type` set to :guilabel:`Australian Employee` and
:guilabel:`Working hours` set to :guilabel:`AU Standard 38 hours/week`. This structure covers all of
the |ATO|'s tax schedules.

If using the **Attendances** or **Planning** app, select the :guilabel:`Work Entry Source` to define
how working hours and days are accounted for on the employee's payslip.

- :guilabel:`Working Schedule`: Work entries are automatically generated based on the employee's
  working schedule, starting from the contract's start date.

  .. example::
     An employee works 38 hours a week, their contract begins on 01/01, today's date is 16/01, and
     the user generates a pay run from 14/01 to 20/01. The working hours on the payslip will be
     automatically calculated to be 38 hours (5 * 7.36 hours) if no unpaid leave is taken.

- Work entries are only generated after clocking in and out of the **Attendances** app. Note that

>   attendances can be imported.

Reviewer: **Attendance**: Work entries are only generated after clocking in and out of the **Attendance** app. Note that attendances can be imported.

> - Work entries are generated from planning shifts in the **Planning** app.

Reviewer: **Planning**: Work entries are generated from planning shifts in the **Planning** app.

.. important::
   Timesheets do not impact work entries in Odoo. If timesheets need to be imported in Odoo, import
   them by navigatig to :menuselection:`Payroll app --> Work Entries --> Work Entries` instead.

Ensure the following fields are configured:

  - Select :guilabel:`Fixed Wage` for full-time and part-time employees.
  - Select :guilabel:`Hourly Wage` for casual workers. This allows for adding a `Casual Loading`
    percentage.

  .. tip::
     Set a default :guilabel:`Wage Type` in the salary :ref:`Structure Type
     <payroll/structure-types>` to configure employees in bulk. If needed, the default can be
     overridden on individual employee records if exceptions are needed.

- :guilabel:`Wage`: Enter the wage, and select the corresponding time period. The following options
  are the **only** pay run frequencies accepted byAustralia: :guilabel:`Daily`, :guilabel:`Weekly`,

>   :guilabel:`Fortnightly` (bi-weekly), :guilabel:`Monthly`, and :guilabel:`Quarterly`.

Reviewer: **Wage**: Enter the wage, and select the corresponding time period. The following options are the only pay run frequencies accepted by Australia: **Daily**, **Weekly**, **Fortnightly** (bi-weekly), **Monthly**, and **Quarterly**.

|STP| software to Odoo.

On the *Import YTD Balances* pop-up window, select :guilabel:`Zeroed Previous Payroll` if the
previous payroll system has been zeroed out, or if there is no previous payroll system. Next, enter
the :guilabel:`Previous BMS ID` in the corresponding field.

.. note::
   :guilabel:`BMS ID` is the *Business Management System Identifier* for your organisation in the
   payroll system. This is important when transitioning from other payroll systems to Odoo, as it
   ensures employees maintain continuous year-to-date records on their payslips and |ATO| Portal.

Enter the :guilabel:`Fiscal Year Start Date` in the corresponding field, then click the
:guilabel:`Transfer and Generate YTD` button.

Importing YTD balances
======================

To import the year-to-date balances, navigate to :menuselection:`Payroll app --> Configuration -->
YTD Opening Balances`.

After importing the YTD balances from the previous software, the |ATO| needs to be notified of the
change of BMS ID and payroll IDs. This step does not apply for strategy 1.

To do so, after importing YTD balances, nabigate to :menuselection:`Payroll app --> Reporting -->

> Single Touch Payroll`, and submit the update event created for this purpose.

Reviewer: To do so, after importing YTD balances, navigate to **Payroll app ‣ Reporting ‣ Single Touch Payroll**, and submit the update event created for this purpose.

Once the journal entry is posted, predefined accounts will impact the company's balance sheet
(PAYGW, wages, and superannuation liabilities) and profit & loss report (wages and superannuation
expenses). In addition, the employee's gross wage and PAYG withholding will update the BAS report
for the relevant period (see Tax Grid: W1 and W2). Accounts can be adjusted to the company's chart
of accounts.

Closing the BAS
---------------

Closing the BAS report now includes payroll sections (W1 to W5)

Most Common:

- **W1**: Total salary, wages and other payments
- **W2**: Amounts withheld from salaries or wages and other payments shown at W1
- **W5**: Total amounts withheld (W2 + W4 + W3).

Special cases

- **W4**: Amounts withheld where no ABN is quoted (NOT a payroll tax! You can record W4 amounts on
  vendor bills by using the No ABN tax)
- **W3**: Other amounts withheld (excluding any amount shown at W2 or W4) To use it tick the box
  *Report in W3* in the employee's record, in the *Payroll* tab.

.. note::
   For Consolidated payroll accounting entries, navigate to :menuselection:`Payroll -->
   Configuration --> Settings` and enable :guilabel:`Batch Account Move Lines`.

> Submit STP fies to the ATO

Reviewer: ### Submit STP files to the ATO

On the Super record, the *Direct Debit Status* will be marked as *Failed*. Either resubmit or cancel
the super transaction

- *What one of the fund payment fails?*

On the Super record lines, the *Fund Payment Status* will be marked as *Failed*. Either resubmit or
cancel the super transaction.

Payroll links to other apps
---------------------------

The following apps integrate with the **Payroll** app, and require specific configurations or
considerations.

Time Off
~~~~~~~~

Logging time off of any kind replaces the employee's work entries. This takes place because time off
types are linked to work entry types.

Navigate to :menuselection:`Time Off app --> Configuration --> Time Off Types`. For each type,
configure the following two fields under the *Payroll* section:

- :guilabel:`Work Entry Type`: Defines which work entry is selected on the *Worked Days* table of
  the payslip.
- :guilabel:`Unused Leave Type`: Choose between :guilabel:`Annual`, :guilabel:`Long Service`, or
  :guilabel:`Personal Leave`. If :guilabel:`Personal Leave` is selected, the remaining leave balance
  for this time off type will *not* show up as an entitlement at the time of termination. Unused

>   leaves of the type :guilabel:`Annual` will include leave loading if the employee is eligible.

Reviewer: **Unused Leave Type**: Choose between **General Leave**, **Long Service**, or **Unpaid at Termination**. If **Unpaid at Termination** is selected, the remaining leave balance for this time off type will not show up as an entitlement at the time of termination.

Australia
=========

.. |YTD| replace:: :abbr:`YTD (Year to Date)`
.. |ATO| replace:: :abbr:`ATO (Australian Taxation Office)`
.. |STP| replace:: :abbr:`STP (Single Touch Payroll)`

The Australian payroll localization covers salary computations for employees, including both
employee and employer payroll taxes. It accounts for all local and national regulations.

Before configuring the Australia localization, refer to the general :doc:`payroll <../../payroll>`
documentation, which includes the basic information for all localizations, as well as all universal
settings and fields.

.. important::
   It is **not** recommended for companies to use the **Payroll** app for the following business
   flows:

   - Income stream types: *Foreign Employment Income*
   - Tax treatment category: *actors & performers*
   - Death benefits reporting

>    - Reporting obligations for :abbr:`WPN (Withholding Payer Number )` (instead of :abbr:`ABN

Reviewer 2: ```suggestion
   - Reporting obligations for :abbr:`WPN (Withholding Payer Number)` (instead of :abbr:`ABN
```

Extra space :)

     (Australian Business Number)`)
   - Allowances subject to a varied rate of withholding (such as cents per kilometer allowance and
     travel allowances)

  `Contact us <https://www.odoo.com/help>`_  to make sure Odoo fits the specific payroll

>   requirements in Australia.

Reviewer 2: ```suggestion
   `Contact us <https://www.odoo.com/help>`_  to make sure Odoo fits the specific payroll
   requirements in Australia.
```

RST formatting -- adding an extra space to keep it within the note format.

General configurations
======================

First, the company must be configured. Navigate to :menuselection:`Settings app --> Users &
Companies --> Companies`. From the list, select the desired company, and ensure the following fields
are configured:

- :guilabel:`Company Name`: Enter the business name in this field.
- :guilabel:`Address`: Complete the full address, including the :guilabel:`City`, :guilabel:`State`,
  :guilabel:`Zip Code`, and :guilabel:`Country`.
- :guilabel:`Trading Name`: If the business has an unregistered alias (businesses could use these
  informally in Australia before May 2012), enter it here.

  .. important::
     In May 2012, the :abbr:`ASIC (Australian Securities and Investments Commission)` required
     businesses to register all names. From November 1, 2025 on, all unregistered trading names are
     retired from the :abbr:`ABR (Australian Business Register)`, and businesses are required to
     register their trading name as a business name.

- :guilabel:`ABN`: Enter the company's eleven-digit :abbr:`ABN (Australian Business Number)` in this
  field. After this is entered, click :guilabel:`Verify ABN`, to ensure the number is correct.

> - :guilabel:`GST registered`: Tick this box if the company is registered for Australian :abbr:`GST

Reviewer 2: ```suggestion
- :guilabel:`GST registered`: Select this checkbox if the company is registered for Australian :abbr:`GST
```

As a general note, we are moving away from the word "tick," as it doesn't necessarily translate well. Instead, try to use "select" or "enable."

- :guilabel:`Email Domain`: Enter the email domain for the company.
- :guilabel:`Color`: Select a color for the company.

.. image:: australia/sw-au.png
   :alt: Australian company form configured for Australia.

Payroll settings
----------------

In addition to configuring the company, some **Payroll** app settings must be configured. Navigate
to :menuselection:`Payroll app --> Configuration --> Settings`, and scroll to the
:guilabel:`Australian Localization` section.

Company Information
~~~~~~~~~~~~~~~~~~~

Configure the following fields in the *Company Information* section:

- :guilabel:`Branch Code`: Enter the six-digit :abbr:`BSB (Bank State Branch)` code. This identifies
  the specific bank and branch for payroll transactions.

  .. tip::
    The *Branch Code* can be found in the |ATO| portal. In Odoo, only the numeric value is relevant.
    For example, `Activity statement 002` would translate in Odoo as 2 (002 = 2).

- :guilabel:`Withholding Payer Number`: If available, enter the eight or nine-digit :abbr:`WPN
  (Withholding Payer Number)` issued by the |ATO|. This is for companies *not* entitled to an
  :abbr:`ABN (Australian Business Number)`.

> - :guilabel:`Registered for Working Holiday Maker`: Tick this box if the company is registered for

Reviewer 2: ```suggestion
- :guilabel:`Registered for Working Holiday Maker`: Select this checkbox if the company is registered for
```

Again, we're moving away from using "tick." Use "select" or "enable" instead.

  the :abbr:`WHM (Working Holiday Maker)` program. This indicates the company is registered with the
  |ATO| to withhold the correct taxes for employees working in Australia with an eligible visa, for
  a period of less than 12 months.

   .. seealso::
      `Employer registration for working holiday makers website
      <https://www.ato.gov.au/businesses-and-organisations/starting-registering-or-closing-a-business/registration-obligations-for-businesses/work-out-which-registrations-you-need/taxation-registrations/employer-registration-for-working-holiday-makers>`_

> - :guilabel:`Registered for PALM Scheme`: Tick this checkbox if the company is registered for the

Reviewer 2: ```suggestion
- :guilabel:`Registered for PALM Scheme`: Select this checkbox if the company is registered for the
```

the question :guilabel:`Are you authorised to register for this service on behalf of your employer?`
Click :guilabel:`Yes` to confirm, then click :guilabel:`Next Step`.

The status changes to :guilabel:`Employer Details`, and displays the contact information for both
the company and the selected payroll responsible. Ensure the :guilabel:`Employer Details` and
:guilabel:`Employee Contact` are correct on the form.

If any information needs to be updated, exit the form and update the company or employee information
in the **Settings** app or **Employees** app, then return to the **Payroll** settings page. Click
the :guilabel:`Start Payroll Onboarding` button, and the pop-up window loads to the
:guilabel:`Employer Details` again.

.. image:: australia/au-employer-details.png
   :alt: The employer and responsible page of the onboarding pop-up window.

When the displayed information is correct, click :guilabel:`Next Step` and the status changes to
:guilabel:`Bank Details`. Using the drop-down menu, select the :guilabel:`Bank Journal` for payroll.
After a journal is selected, the :guilabel:`Bank Name`, :guilabel:`Account Name`, :guilabel:`Bank

> Account Number`, and  :guilabel:`BSB` fields are populated with the selected bank journal's details,

Reviewer 2: ```suggestion
Account Number`, and :guilabel:`BSB` fields are populated with the selected bank journal's details,
```

and **cannot** be modified. If any changes are needed, exit the form and update the information for
the journal in the **Accounting** app.

.. image:: australia/au-bank-details.png
   :alt: The bank info page of the onboarding pop-up window.

When the :guilabel:`Bank Details` are correctly filled out, click :guilabel:`Next Step`, and the
status changes to :guilabel:`Signature`. Two PDFs appear, :guilabel:`Odoo Terms & Conditions`, and

> :guilabel:`SuperChoice FSG PDS DDA`. Download and read the two PDFs, then tick the two checkboxes

Reviewer 2: ```suggestion
:guilabel:`SuperChoice FSG PDS DDA`. Download and read the two PDFs, then select the two checkboxes
```

:guilabel:`Working hours` set to :guilabel:`AU Standard 38 hours/week`. This structure covers all of
the |ATO|'s tax schedules.

If using the **Attendances** or **Planning** app, select the :guilabel:`Work Entry Source` to define
how working hours and days are accounted for on the employee's payslip.

- :guilabel:`Working Schedule`: Work entries are automatically generated based on the employee's
  working schedule, starting from the contract's start date.

  .. example::
     An employee works 38 hours a week, their contract begins on 01/01, today's date is 16/01, and
     the user generates a pay run from 14/01 to 20/01. The working hours on the payslip will be
     automatically calculated to be 38 hours (5 * 7.36 hours) if no unpaid leave is taken.

- :guilabel:`Attendances`: Work entries are only generated after clocking in and out of the
  **Attendances** app. Note that attendances can be imported.
- :guilabel:`Planning`: Work entries are generated from planning shifts in the **Planning** app.

.. important::
   Timesheets do not impact work entries in Odoo. If timesheets need to be imported in Odoo, import

>    them by navigatig to :menuselection:`Payroll app --> Work Entries --> Work Entries` instead.

Reviewer 2: ```suggestion
   them by navigating to :menuselection:`Payroll app --> Work Entries --> Work Entries` instead.
```

Spelling :)

Schedule section
~~~~~~~~~~~~~~~~

- :guilabel:`Work Entry Source`: Defines how :doc:`work entries <../work_entries>` are generated for
  payroll during the specified pay period. The options are:

  - :guilabel:`Working Schedule`: Based on the employee's assigned :ref:`working schedule
    <employees/schedule>` (e.g., 40 hours per week).
  - :guilabel:`Attendances`: Based on :doc:`approved checked-in hours
    <../../attendances/management>` in the **Attendances** app.
  - :guilabel:`Planning`: Based on :ref:`scheduled shifts <planning/shifts>` in the **Planning**
    app.

- :guilabel:`Working Hours`: Using the drop-down menu, select the default work schedule. This is
  particularly important for employees available to receive overtime pay (typically hourly
  employees, not salaried). The common selection is :guilabel:`AU Standard 38 hours/week`.

General section
~~~~~~~~~~~~~~~

- :guilabel:`Regular Pay Day`: Day of a week to process the payments if relevant.
- :guilabel:`Report in BAS - W3`: Enable this to add PAYG withholding amounts in BAS section W3
  instead of W2. Refer to the `ATO's web page on PAYG withholding

>   <https://www.ato.gov.au/businesses-and-organisations/preparing-lodging-and-paying/business-activity-statements-bas/in-detail/instructions/payg-withholding-how-to-complete-your-activity-statement-labels#W3Otheramountswithheldexcludinganyamount>`_

Reviewer 2: This link 404s for me. You may need to find a more current one.

  for more information.

Personal information section
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

- :guilabel:`Non-resident`: Check this box if the employee is not a legal resident of Australia.
- :guilabel:`Tax-free Threshold`: Check this box if the employee is claiming tax-free threshold.

> - :guilabel:`TFN Status`: Using the drop-down menu, select the eployee's declaration.

Reviewer 2: ```suggestion
- :guilabel:`TFN Status`: Using the drop-down menu, select the employee's declaration.
```

  Odoo.

Tax and contributions sections
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

- :guilabel:`Employment Type`: Select the kind of employment the employee has. This affects tax
  calculations. Options are :guilabel:`Full time`, :guilabel:`Part time`, :guilabel:`Casual`,
  :guilabel:`Labour hire`, :guilabel:`Voluntary agreement`, :guilabel:`Death beneficiary`, or
  :guilabel:`Non-employee.`
- :guilabel:`Income Stream Type`: Select where the employee is receiving their pay from. The typical
  employee is :guilabel:`Salary and wages`, however some other circumstances may require a different
  selection.
- :guilabel:`Tax Treatment Category`: The |STP| Phase 2 report includes a 6-character tax treatment
  code for each employee. The tax treatment code is an abbreviated way of explaining the factors
  that can influence the amount withheld from employees. The |STP| solution automates the reporting
  of these codes and ensure that the tax treatment code reported is valid. Even though the creation
  of this code is automated, it is still part of the |STP| report, and it is important to understand
  what it means. For more details, check the |ATO| website on `how to report employment and tax

>   information through STP Phase 2 <https://www.ato.gov.au/businesses-and-organisations/hiring-and-paying-your-workers/single-touch-payroll/in-detail/single-touch-payroll-phase-2-employer-reporting-guidelines/how-to-report-employment-and-tax-information-through-stp-phase-2>`_.

Reviewer 2: ```suggestion
  information through STP Phase 2
  <https://www.ato.gov.au/businesses-and-organisations/hiring-and-paying-your-workers/single-touch-payroll/in-detail/single-touch-payroll-phase-2-employer-reporting-guidelines/how-to-report-employment-and-tax-information-through-stp-phase-2>`_.
```

- :guilabel:`HELP / STSL`: Click this checkbox if the employee is a :abbr:`HELP (Higher Education

>   Loan Program` loan or :abbr:`STSL (Study Training Support Loan)` recipient.

Reviewer 2: ```suggestion
  Loan Program)` loan or :abbr:`STSL (Study Training Support Loan)` recipient.
```

- :guilabel:`Medicare Variation Form` and :guilabel:`Medicare levy surcharge`: Click
  :guilabel:`Upload your file` to upload the filled form and select the appropriate surcharge if
  applicable.

Deductions, offsets, and withholding section
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

- :guilabel:`Child Support Deduction`: The amount that has to be deducted every pay period, subject
  to :abbr:`PEA (Protected Earnings Amount)`.
- :guilabel:`Child Support Garnishee Amount %`: The percentage of pay that is legally withheld and
  paid towards child support.
- :guilabel:`Withhold for Extra Pay`: An amount that has to be deducted every pay period, subject to
  :abbr:`PEA (Protected Earnings Amount)`.
- :guilabel:`Withholding Variation`: A formal adjustment to the standard tax withholding rate based
  on specific circumstances (e.g. financial hardship or |ATO| approval).
- :guilabel:`Additional Withholding Amount`:Additional amount will be withheld from the employee's

>   salary after PAYG withholding (Schedule 14)5.

Reviewer 2: ```suggestion
- :guilabel:`Additional Withholding Amount`: Additional amount will be withheld from the employee's
  salary after PAYG withholding (Schedule 14)5.
```

Related question for line 374: Is it "Schedule (14)5" or "Schedule (145)"?

Leave loading / workplace giving section
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> - :guilabel:`Eligible for Leave Loading`: Check if the employee is eligible for leabe loading.

Reviewer 2: ```suggestion
- :guilabel:`Eligible for Leave Loading`: Check if the employee is eligible for leave loading.
```

- :guilabel:`Workplace Giving Employee`: Set the amount to be deducted.
- :guilabel:`Salary Sacrificed Workplace Giving`: Set the amount to be salary sacrificed (e.g.,
  receiving a benefit instead of a deduction).

Super contributions section
~~~~~~~~~~~~~~~~~~~~~~~~~~~

- :guilabel:`Extra Negotiated Super %`: Set the additional percentage amount to be contributed on
  top of the super guarantee.
- :guilabel:`Extra Compulsory Super %`: Set the additional percentage amount to be contributed as
  per industrial agreements or awards obligations.

Salary sacrifice section
~~~~~~~~~~~~~~~~~~~~~~~~

- :guilabel:`Salary Sacrifice Superannuation`: Allows employees to sacrifice part of their salary in
  favour of :abbr:`RESC (Reportable Employer Superannuation Contributions)` account, which is an
  Australian retirement account.
- :guilabel:`Salary Sacrifice Other Benefits`: Allows employees to sacrifice part of their salary
  towards some other form of benefit. Refer to the `ATO's web page on Salary sacrificing for

>   employees <https://www.ato.gov.au/individuals-and-families/jobs-and-employment-types/working-as-an-employee/salary-sacrificing-for-employees>`_

Reviewer 2: ```suggestion
  employees
  <https://www.ato.gov.au/individuals-and-families/jobs-and-employment-types/working-as-an-employee/salary-sacrificing-for-employees>`_
```

configured.

Navigate to :menuselection:`Contacts app --> Configuration --> Bank Accounts`. Click :guilabel:`New`
to add a bank account, and a *Bank Accounts* form loads.

Configure the form, ensuring the following fields are populated:

- :guilabel:`Account Number`: Enter the bank account number in this field.
- :guilabel:`Clearing Number`: Enter the clearing number in this field.
- :guilabel:`BSB`: Enter the Bank State Branch code in this field.
- :guilabel:`Account Holder`: Using the drop-down menu, select the corresponding **Contact** app
  record for the account holder, typically the company.
- :guilabel:`Account Holder Name`: The selected :guilabel:`Account Holder` populates this field by
  default. Make any necessary modifications.
- :guilabel:`Bank`: Using the drop-down menu, select the bank the account is under. Once selected, a
  *Bank Information* tab appears, containing the various bank information. Make any desired edits to
  this tab.
- :guilabel:`Send Money`: Ensure this toggle is set to :guilabel:`Trusted`.
- :guilabel:`Currency`: Set this field to :guilabel:`AUD`.

.. image:: australia/bank-au.png

>    :alt: Add alt text.

Reviewer 2: This line needs alt text.

Then, navigate to :menuselection:`Accounting app --> Configuration --> Journals`, and select the
corresponding bank journal. Ensure the :guilabel:`Bank Account` number in the *Journal Entries* tab
is the same as the bank account configured above.

.. important::
   Odoo takes direct debits from this account to pay employee's superannuation.

.. _payroll/payroll_localizations/au_config:

Payroll configuration
=====================

Several sections within the **Payroll** app also install a salary structure, structure type, rules,
and parameters specific to Australia.

Salary structures & structure types
-----------------------------------

When the **l10n_au_hr_payroll** module is :ref:`installed <payroll/au_apps>`, a new
:guilabel:`Salary Structure` gets installed, :guilabel:`Australian Employee`. This structure

> includes one :guilabel:`Structure Type`, :guilabel:`Australian Employee  Pay`.

Reviewer 2: ```suggestion
includes one :guilabel:`Structure Type`, :guilabel:`Australian Employee Pay`.
```

percentage) and the cap (a dollar amount).

To view rule parameters, navigate to :menuselection:`Payroll app --> Configuration --> Rule
Parameters`. Here, all rule parameters are displayed with their linked :guilabel:`Salary Rules`,
which can be accessed. Review the parameters associated with a rule by looking for the
:guilabel:`Name` of the rule, and make any edits as needed.

.. important::
   Odoo adds updated rule parameters for the current calendar year. It is **not** recommended to
   edit rule parameters **unless a national or local parameter has changed**, and is different from
   the rule parameters created by Odoo. Check with all local and national regulations *before*
   making any changes to rule parameters.

Payroll onboarding
==================

Once the :ref:`company <payroll/payroll_localizations/au_gen>`, the :ref:`STP responsible employee
<payroll/payroll_localizations/stp>`, and the :ref:`bank journal for direct debit
<payroll/payroll_localizations/bank_journals>` are configured, navigate to :menuselection:`Payroll
app --> Configurations --> Settings` to start the onboarding process.

> Scroll ot the *Australian Localization* section, and under *SuperStream and Single Touch Payroll*,

Reviewer 2: ```suggestion
Scroll to the *Australian Localization* section, and under *SuperStream and Single Touch Payroll*,
```

select the :guilabel:`Payroll Mode`. Choose :guilabel:`Production` for a production database with a
valid subscription code, or otherwise, select :guilabel:`Testing`. Then, click the :guilabel:`Start
Payroll Onboarding` button, and a *Payroll Onboarding* pop-up window loads.

.. important::
   It is strongly recommended to test payroll on the testing database prior to configuring the
   production environment, as any payroll processed in the production database will be submitted
   directly to the |ATO|.

On the *Payroll Responsible* stage, select the :ref:`STP responsible employee
<payroll/payroll_localizations/stp>` in the :guilabel:`Payroll Responsible` field, then click the
:guilabel:`Next Step` button.

On the *Authorised* stage, click the :guilabel:`Yes` option for the question :guilabel:`Are you

> authorised to register for this service on behalf of your employer`, then click the :guilabel:`Next

Reviewer 2: Is there a question mark at the end of this guilabel? Just checking.

   adjusted slightly (with a few field changes), and re-imported as work entries.

.. important::
   Currently there is **no** integration with Odoo's **Timesheets** app.

Generate payslips
-----------------

Payslips can be generated in two ways:

- :ref:`Pay Runs <payroll/payroll_localizations/au_run>`: The recommended method for Australia.
- Off-Cycle Payslips: Mainly used for termination payslips, or other payments outside of a pay
  cycle.

There is no limit to the amount of payslips that can be created in one pay run.

After creating a :ref:`pay run <payroll/payroll_localizations/au_run>`, one payslip is created per
employee in the Waiting stage, which can be reviewed and amended before validation.

.. note::
   In the future, payslips that are not related to hours worked should also be handled using
   off-cycle payslips.

Payslips
--------

On the payslip form view, there are two types of inputs:

- :guilabel:`Worked days`: Computed based on the work entry source set on the employee's contract.
  Work entries can be configured according to different types: attendance, overtime, Saturday rate,
  Sunday rate, public holiday rate, etc.

> - :guilabel:`Salary  inputs`: Individual payments or amounts of different types (allowances, lump

Reviewer 2: ```suggestion
- :guilabel:`Salary inputs`: Individual payments or amounts of different types (allowances, lump
```

     salary and remitted to the ATO.
   - Net Salary: The "take-home pay" or the employee's actual wage after all taxes and deductions.
   - Concessional Super Contribution: In this scenario, the amount sacrificed to superannuation,
     payable to the employee's super fund in addition to the super guarantee
   - Super Guarantee: This is computed as 12% of the ordinary time earnings amount

Confirming payslips
-------------------

Once all payslip data is deemed correct, click :guilabel:`Create Draft Entry` on the pay run. This
can also be done payslip by payslip for control reasons.

This has several impacts:

- The employee receives their payslip via email + portal accessibility.
- Marking the batch and its payslips as Done.
- Creating a draft accounting entry per payslip or one entry for the whole batch, depending on the
  payroll settings. At this stage, accountants can post entries to affect the balance sheet, P&L
  report, and BAS report.
- Preparing the |STP| submission (or payroll data to be filed to the ATO as part of |STP|
  compliance). This needs to be performed by the STP Responsible user, defined under

>   :menuselection:`Payroll --> Configuration--> Settings`.

Reviewer 2: ```suggestion
  :menuselection:`Payroll --> Configuration --> Settings`.
```

- Preparing super contribution lines as part of SuperStream compliance. This needs to be done by the

>   HR Super Send user selected under :menuselection:`Payroll --> Configuration--> Settings`

Reviewer 2: ```suggestion
  HR Super Send user selected under :menuselection:`Payroll --> Configuration --> Settings`
```

.. note::
   The following salary rules that impact accounting:

   - Fees and Deductions
   - Total Withholding
   - Child Support Total
   - Expense Reimbursement
   - Net Salary
   - Concessional Super Contribution
   - Super Guarantee

Once the journal entry is posted, predefined accounts will impact the company's balance sheet
(PAYGW, wages, and superannuation liabilities) and profit & loss report (wages and superannuation
expenses). In addition, the employee's gross wage and PAYG withholding will update the BAS report
for the relevant period (see Tax Grid: W1 and W2). Accounts can be adjusted to the company's chart
of accounts.

Closing the BAS
---------------

Closing the BAS report now includes payroll sections (W1 to W5)

Most Common:

- **W1**: Total salary, wages and other payments
- **W2**: Amounts withheld from salaries or wages and other payments shown at W1
- **W5**: Total amounts withheld (W2 + W4 + W3).

Special cases

- **W4**: Amounts withheld where no ABN is quoted (NOT a payroll tax! You can record W4 amounts on
  vendor bills by using the No ABN tax)
- **W3**: Other amounts withheld (excluding any amount shown at W2 or W4) To use it tick the box

>   *Report in W3* in the employee's record, in the *Payroll* tab.

Reviewer 2: ```suggestion
- **W3**: Other amounts withheld (excluding any amount shown at W2 or W4). To use it, select the
  *Report in W3* checkbox in the employee's record in the *Payroll* tab.
```

.. tip::
   A warning message appears if important information is missing.

On the |STP| record for the pay run, a few useful information is displayed:

- A warning message if important information is missing.
- An automatically generated activity for the |STP| responsible user.
- A summary of payslips contained in this pay run, auditable from this view.

Always test the submission before submitting.

Once the |STP| record is ready, click :guilabel:`Submit to ATO`, then read and accept the related
terms and conditions.

.. image:: australia/payroll-stp-record.png
   :alt: Example of an STP record.

Pay, refund, and print payslips
-------------------------------

When payslips are submitted to |ATO| you will see 2 options:

- :guilabel:`Mark as Paid`: Click this if you are not using Odoo for accounting (this serves as an
  internal note that the pay run was paid).
- :guilabel:`Paid`: Click this if you want to generate a payment file for later reconciliation.
  Here, you can also select :guilabel:`ABA Credit Transfer` as the payment method. The ABA file can
  be downloaded directly from the pay run.

 .. seealso::
   - :ref:`ABA files for batch payments <australia/aba>`

>    - :ref:`How to pay, refund and print payslips <payroll/new-payslip>`

Reviewer 2: ```suggestion
.. seealso::
   - :ref:`ABA files for batch payments <australia/aba>`
   - :ref:`How to pay, refund and print payslips <payroll/new-payslip>`
```

   - Fund payment status (payment status from the clearing house to the super funds)
   - Payment status (payment status of the entire transaction)

#. Reconcile. You can now reconcile the payment with the bank statement.

.. note::
   As of July 1, 2026, Australian businesses will be **required** to pay super at the same time as
   salary & wages. This Governmental initiative is called *Payday Super*. Find more information on
   the `ATO website
   <https://www.ato.gov.au/businesses-and-organisations/super-for-employers/payday-super>`_ .

Special cases
-------------

The following are common situations and their resolutions:

- *What if I need to cancel my super payment?*

Navigate to :menuselection:`Payroll app --> Reporting --> Super Contributions`. Find the entry to
cancel and click :guilabel:`Cancel`. The chatter log will display information regarding the *Direct
Debit Status*, identifying whether it is a cancellation or a refund.

- *What if my direct debit payment fails?*

On the Super record, the *Direct Debit Status* will be marked as *Failed*. Either resubmit or cancel
the super transaction

> - *What one of the fund payment fails?*

Reviewer 2: ```suggestion
- *What if one of the fund payment fails?*
```

.. _payroll/au-terminate:

Terminate employees
-------------------

Employees can be terminated by going to :menuselection:`Payroll app --> Reporting --> Terminate
Employee`. The following fields must be completed:

- :guilabel:`Contract End Date`: Once the termination is validated, this date is added to the
  contract automatically, and marks the contract as *Expired* when the date has been reached.
- :guilabel:`Cessation Type Code`: A mandatory field for the ATO's STP reporting.
- :guilabel:`Termination Type`: The type of redundancy (genuine or non-genuine) affects the
  computation of unused annual and long service leave withholding.

The Final payslip will be marked as final pay. Besides normal wages & benefits, it will contain
unused leaves, auto-calculated by Odoo.

On top of these payments, you can add ETPs. These can be of 2 types: Excluded ETPs & Non-excluded
ETPs. These are simply taxed differently, and cover different types of employment scenarios. They
also depend on the type of redundancy. Find more information on the `ATO website
<https://www.ato.gov.au/businesses-and-organisations/super-for-employers/payday-super>`_ .

.. note::

>    Annual leaves will be taxed as per as per |ATO| schedule 7. ETPs will be taxed as per |ATO|

Reviewer 2: ```suggestion
   Annual leaves will be taxed as per |ATO| schedule 7. ETPs will be taxed as per |ATO|
```

  a :guilabel:`OTE` (one-time earning), :guilabel:`Salary & Wages`, or :guilabel:`Not Salary &
  Wages`.
- :guilabel:`STP Code`: This field is only visible in developer mode. This field tells Odoo how to
  report the gross value of the payment to the |ATO|. It is **not** recommended to change the value
  of this field if it is already populated by default.
- :guilabel:`Is OTE`: This field determines whether time spent in this category can be considered
  ordinary time earnings, meaning that the superannuation guarantee rate will apply (e.g., regular
  attendance, paid leave, etc.).
- :guilabel:`Penalty Rate`: This field is used to determine the percentage of penalty that applies
  to time spent in this category. It is important to configure the penalty rate that applies in the
  state or industry according to the type of work (e.g., Saturday rate, Sunday rate, overtime rate,
  etc.).

Close Payroll
-------------

If there are no errors, payroll is completed for the pay period.

Reports
=======

The Australian localization contains several reports unique to Australia, which provide information

> on superfund contributions, single touch payroll, :abbr:`STP (Single Touch Payroll)`  Finalisation,

Reviewer 2: ```suggestion
on superfund contributions, single touch payroll, :abbr:`STP (Single Touch Payroll)` Finalisation,
```

and the ability to terminate employees.

STP Finalisation
----------------

The fiscal year in Australia ends on June 30th, but :abbr:`STP (Single Touch Payroll)` finalisation
declarations are due by July 14th. These can be done in :ref:`one single report for all employees
<payroll/stp-total>`, or :ref:`individually <payroll/stp-individual>`.

.. _payroll/stp-total:

EOFY finalisation
~~~~~~~~~~~~~~~~~

To finalize the fiscal year's :abbr:`STP (Single Touch Payroll)`, navigate to
:menuselection:`Payroll app --> Reporting --> STP Finalisation`, and a :guilabel:`Finalise Payroll`
pop-up window loads.

The :guilabel:`Company`, :guilabel:`ABN`, and :guilabel:`Branch Code` are populated from the
:ref:`company information <payroll/payroll_localizations/au_gen>`.

Configure the following fields on the form:

- :guilabel:`BMS ID`: Enter the company's :abbr:`BMS ID (Business Management Software Identifier)`,
  issued by the |ATO|.

> - :guilabel:`EOFY Declaration`: Tick this checkbox to indicate the finalisation is for the entire

Reviewer 2: ```suggestion
- :guilabel:`EOFY Declaration`: Select this checkbox to indicate the finalisation is for the entire
```

>   fiscal year (not an :ref:`individual finalisation <payroll/stp-individual>`). Once ticket, the

Reviewer 2: ```suggestion
- :guilabel:`EOFY Declaration`: Select this checkbox to indicate the finalisation is for the entire
  fiscal year (not an :ref:`individual finalisation <payroll/stp-individual>`). Once selected, the
```

  name of the report changes to `EOFY Finalisation - YYYY/YY`.

> - :guilabel:`Finalisation`: Tick this checkbox to indicate this is a finalisation and not an

Reviewer 2: ```suggestion
- :guilabel:`Finalisation`: Select this checkbox to indicate this is a finalisation and not an
```

  amendment.
- :guilabel:`Date Start`: This field displays `1 July`, by default.

> - :guilabel:`Date End`: If :guilabel:`EOFY Declaration` is ticked, this field displays `30 June

Reviewer 2: ```suggestion
- :guilabel:`Date End`: If :guilabel:`EOFY Declaration` is selected, this field displays `30 June
```

- :guilabel:`Responsible User`: Using the drop-down menu, select the user responsible for this
  report.

.. image:: australia/au-stp-eofy.png
   :alt: EOFY finalisation top half filled out.

Both active and :ref:`terminated <payroll/au-terminate>` employees to finalise are displayed in the
:guilabel:`Employee Finalisation` tab. Ensure all balances for all employees are correct. Once
verified, click :guilabel:`Submit to ATO` to create the report and submit it.

Employee payment information changes to *Tax ready* on their online income statement, after the end
of the financial year.

.. _payroll/stp-individual:

Individual finalisation
~~~~~~~~~~~~~~~~~~~~~~~

Employees can be individually finalised during the year. This can be useful when:

- A one-off (off-cycle) payment is made *after* a finalisation is sent to the :abbr:`ATO (Australian
  Taxation Office)`.
- After an employee is :ref:`terminated <payroll/au-terminate>` during the year.

To create an individual finalisation, navigate to :menuselection:`Payroll app --> Reporting --> STP
Finalisation`.

Configure the top-half of the form in the same manner as the :ref:`typical annual finalization

> <payroll/stp-total>`, except **do not** tick the checkbox next to the :guilabel:`EOFY Declaration`.

Reviewer 2: ```suggestion
<payroll/stp-total>`, but **do not** select the checkbox next to the :guilabel:`EOFY Declaration`.
```

====================================

To ensure |STP| compliance, the |ATO| requires Digital Service Providers (DSPs) like Odoo to adhere
to strict security standards. The following requirements have been implemented within the Odoo
ecosystem:

#. Audit Logging: Odoo maintains comprehensive audit logs for all payroll and accounting-related
   records. These logs are stored in the database and synchronized with our IAP server for a rolling
   12-month period.

#. Mandatory MFA: Multi-Factor Authentication (MFA) is mandatory for all users with access to
   payroll and accounting records. This security feature cannot be disabled.

#. Authentication Persistence: The "Remember Me" functionality is limited to a maximum of 24 hours
   after 2FA authentication. Once this period expires, Odoo will automatically log the user out and
   require a fresh 2FA login.

#. Idle Session Timeout: Sessions will automatically time out after 30 minutes of inactivity.

#. Brute Force Protection: User accounts will be locked after a maximum of 5 unsuccessful login
   attempts.

#. Onshore Data Hosting: By default, all payroll and accounting records must be hosted onshore. If

>    data is hosted offshore, the customer or partner must proactively contact the |ATO|'s' Digital

Reviewer 2: ```suggestion
   data is hosted offshore, the customer or partner must proactively contact the |ATO|'s Digital
```

Australia
=========

.. |YTD| replace:: :abbr:`YTD (Year to Date)`
.. |ATO| replace:: :abbr:`ATO (Australian Taxation Office)`

The Australian payroll localization covers salary computations for employees, including both
employee and employer payroll taxes. It accounts for all local and national regulations.

Before configuring the Australia localization, refer to the general :doc:`payroll <../../payroll>`
documentation, which includes the basic information for all localizations, as well as all universal
settings and fields.

.. important::

>    As of Odoo 19, it is **not** recommended for companies to use the **Payroll** app for the

Reviewer 3: Remember to remove the direct reference to the version in your forward ports, after 19.0! We don't want to reference the version directly in the text after

"As of Odoo 19, it.." --> "It..."

   following business flows:

   - Income stream types: *Foreign Employment Income*
   - Tax treatment category: *actors & performers*
   - Death benefits reporting

>    - Reporting obligations for WPN (instead of ABN)

Reviewer 3: Add `abbr` tags for WPN and ABN

   :header-rows: 1

   * - Name
     - Technical name
     - Dependencies
     - Description
   * - :guilabel:`Australia - Payroll`
     - `l10n_au_hr_payroll`
     - - hr_payroll
       - hr_work_entry_holidays
       - hr_payroll_holidays
       - base_address_extended
     - Provides Australian payroll basics, including employee tax details, salary structures
       (Basic/Gross/Net), and tax rules.
   * - :guilabel:`Australia - Payroll with Accounting`
     - `l10n_au_hr_payroll_account`
     - - l10n_au_hr_payroll
       - hr_payroll_account
       - l10n_au
       - l10n_au_aba
     - Links payroll and accounting by creating journal entries (per payslip if needed) to record
       payroll in the company's books.
   * - :guilabel:`Australia - Payroll with API`
     - `l10n_au_hr_payroll_api`
     - - l10n_au_hr_payroll_account
       - account_edi_proxy_client
       - auth_timeout
     - Provides STP and Super Stream compliance via the Superchoice API, with ATO-approved security
       controls including MFA, session timeouts, and audit logging.

.. seealso::
   :doc:`Configure the Australia fiscal localization
   <../../../finance/fiscal_localizations/australia>`

> .. _payroll/au_gen:

Reviewer 3: custom anchor style

```suggestion
.. _payroll/payroll_localizations/au_gen:
```

Companies --> Companies`. From the list, select the desired company, and ensure the following fields
are configured:

- :guilabel:`Company Name`: Enter the business name in this field.
- :guilabel:`Address`: Complete the full address, including the :guilabel:`City`, :guilabel:`State`,
  :guilabel:`Zip Code`, and :guilabel:`Country`.
- :guilabel:`Trading Name`: If the business has an unregistered alias (businesses could use these
  informally in Australia before May 2012), enter it here.

  .. important::
     In May 2012, the :abbr:`ASIC (Australian Securities and Investments Commission)` required
     businesses to register all names. From November 1, 2025 on, all unregistered trading names are
     retired from the :abbr:`ABR (Australian Business Register)`, and businesses are required to
     register their trading name as a business name.

- :guilabel:`ABN`: Enter the company's eleven-digit :abbr:`ABN (Australian Business Number)` in this
  field. After this is entered, click :guilabel:`Verify ABN`, to ensure the number is correct.
- :guilabel:`GST registered`: Tick this box if the company is registered for Australian :abbr:`GST
  (Goods and Services Tax)`.

> - :guilabel:`ACN` Enter the company's nine-digit :abbr:`ACN (Australian Company Number)`. This

Reviewer 3: ```suggestion
- :guilabel:`ACN`: Enter the company's nine-digit :abbr:`ACN (Australian Company Number)`. This
```

- :guilabel:`Currency`: By default, :abbr:`AUD (Australian Dollars)` is selected. If not, select
  :guilabel:`AUD` from the drop-down menu.
- :guilabel:`Phone`: Enter the company phone number.
- :guilabel:`Email`: Enter the email used for general contact information.
- :guilabel:`Website`: Enter the company's web address.
- :guilabel:`Email Domain`: Enter the email domain for the company.
- :guilabel:`Color`: Select a color for the company.

.. image:: australia/sw-au.png
   :alt: Australian company form configured for Australia.

Payroll settings
----------------

In addition to configuring the company, some **Payroll** app settings must be configured. Navigate
to :menuselection:`Payroll app --> Configuration --> Settings`, and scroll to the
:guilabel:`Australian Localization` section.

Company Information
~~~~~~~~~~~~~~~~~~~

Configure the following fields in the :guilabel:`Company Information` section:

- :guilabel:`Branch Code`: Enter the six-digit :abbr:`BSB (Bank State Branch)` code. This identifies
  the specific bank and branch for payroll transactions.
- :guilabel:`Withholding Payer Number`: If available, enter the eight or nine-digit :abbr:`WPN

>   (Withholding Payer Number)` issued by the ATO. This is for companies *not* entitled to an

Reviewer 3: ```suggestion
  (Withholding Payer Number)` issued by the |ATO|. This is for companies *not* entitled to an
```

  :abbr:`ABN (Australian Business Number)`.
- :guilabel:`Registered for Working Holiday Maker`: Tick this box if the company is registered for
  the :abbr:`WHM (Working Holiday Maker)` program. This indicates the company is registered with the

>   ATO to withhold the correct taxes for employees working in Australia with an eligible visa, for a

Reviewer 3: ```suggestion
  |ATO| to withhold the correct taxes for employees working in Australia with an eligible visa, for a
```

  period of less than 12 months.
- :guilabel:`Registered for PALM Scheme`: Tick this checkbox if the company is registered for the
  :abbr:`PALM (Pacific Australia Labour Mobility)` scheme, which allows the company to hire
  employees from ten different Pacific island nations.

SuperStream and Single Touch Payroll
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Configure the following fields in the :guilabel:`SuperStream and Single Touch Payroll` section:

- :guilabel:`Payroll Mode`: Using the drop-down menu, select whether SuperStream and :abbr:`STP
  (Single Touch Payroll)` are in :guilabel:`Testing` mode, or :guilabel:`Production` (live) mode.
- :guilabel:`Registration Status`: This field is unable to be modified, as it displays the current
  registration status for the SuperStream and :abbr:`STP (Single Touch Payroll)` payroll.

.. important::
   Odoo is currently compliant with :guilabel:`SuperStream and Single Touch Payroll` Phase 2 and
   SuperStream. However, since this is relatively new, it is recommended the company contacts Odoo's
   sales or support teams *prior to* registration in **Production** mode, to ensure the

>    configuration suits the business's needs.

Reviewer 3: this framing somewhat casts doubt in our software, even though i understand what you're trying to say here is that this needs to be super accurate and there is implementation risk. We can slightly tweak this to make this clear (: 

Odoo is compliant with SuperStream and Single Touch Payroll Phase 2. Before registering in Production mode, we recommend reviewing your configuration with Odoo’s sales or support teams to ensure it aligns with your company’s specific payroll requirements.

 
```suggestion
   Odoo is compliant with :guilabel:`SuperStream and Single Touch Payroll` Phase 2. Before registering in **Production** mode, we recommend reviewing your configuration with Odoo's sales or support teams [optional TODO: links to sales or support] to ensure it aligns with your company’s specific payroll requirements.
```

Reviewer 4: Hi @Felicious 

Definitely looks better. The only feedback here is that Australian payroll regulations include a number of edge cases that Odoo does not currently cover at this stage. So ideally, if the customer has any doubts, it is better for them to contact us.

So, something more like:

Odoo is compliant with SuperStream and Single Touch Payroll Phase 2. Before registering in Production mode, we recommend consulting with Odoo’s sales or support teams to ensure it aligns with your company’s specific payroll requirements.

Schedule section
~~~~~~~~~~~~~~~~

- :guilabel:`Work Entry Source`: Defines how :doc:`work entries <../work_entries>` are generated for
  payroll during the specified pay period. The options are:

  - :guilabel:`Working Schedule`: Based on the employee's assigned :ref:`working schedule
    <employees/schedule>` (e.g., 40 hours per week).
  - :guilabel:`Attendances`: Based on :doc:`approved checked-in hours
    <../../attendances/management>` in the **Attendances** app.
  - :guilabel:`Planning`: Based on :ref:`scheduled shifts <planning/shifts>` in the **Planning**
    app.

- :guilabel:`Working Hours`: Using the drop-down menu, select the default work schedule. This is
  particularly important for employees available to receive overtime pay (typically hourly
  employees, not salaried). The common selection is :guilabel:`AU Standard 38 hours/week`.

Deductions, offsets, and withholding section
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

- :guilabel:`Child Support Deduction`: Enter the amount due each pay period for child support.
- :guilabel:`Child Support Garnishee Amount %`:
- :guilabel:`Withhold for Extra Pay`:
- :guilabel:`Withholding Variation`:

> - :guilabel:`Additional Withholding Amount`:

Reviewer 3: looks like these definitions are missing explanations!

Reviewer 4: @Felicious 

Child Support Garnishee Amount %: The percentage of pay that is legally withheld and paid towards child support.
Withhold for Extra Pay: An additional amount of tax withheld specifically for "irregular" payments - such as bonuses, commissions, or back pay
Withholding Variation: A formal adjustment to the standard tax withholding rate based on specific circumstances (e.g. financial hardship or ATO approval).
Additional Withholding Amount: An extra fixed amount voluntarily withheld from pay on top of normal tax.

Super contributions section
~~~~~~~~~~~~~~~~~~~~~~~~~~~

- :guilabel:`Extra Negotiated Super %`: Enter the percentage the company pays *in addition to* the
  government mandated minimum into the employee's :abbr:`RESC (Reportable Employer Superannuation
  Contributions)` (retirement) account.

> - :guilabel:`Extra Compulsory Super %`:

Reviewer 3: missing definition here!

General section
~~~~~~~~~~~~~~~

- :guilabel:`Regular Pay Day`: Select the day of the week the employee is paid using the drop-down
  menu.

> - :guilabel:`Report in BAS - W3`:

Reviewer 3: missing explanation!

Personal information section
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

- :guilabel:`Non-resident`: Tick this checkbox if the employee is **not** a legal resident of
  Australia.
- :guilabel:`Tax-free Threshold`: If the employee **is** a legal Australian resident, this field
  appears.
- :guilabel:`TFN Status`: Using the drop-down menu, select the :abbr:`TFN (Tax Filing Number)`
  status of the employee. All options aside from :guilabel:`Declaration not completed...` are

>   treated as if a :abbr:`TFN (Tax Filing Number)` is provided

Reviewer 3: ```suggestion
  treated as if a :abbr:`TFN (Tax Filing Number)` is provided.
```

- :guilabel:`Tax File Number`: If :guilabel:`Declaration provided` is selected for the
  :guilabel:`TFN Status` field, enter the tax filing number in the field. If another option was
  selected, this field is populated with a placeholder number, and **cannot** be modified.

Tax and contributions sections
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

- :guilabel:`Employment Type`: Select the kind of employment the employee has. This affects tax
  calculations. Options are :guilabel:`Full time`, :guilabel:`Part time`, :guilabel:`Casual`,
  :guilabel:`Labour hire`, :guilabel:`Voluntary agreement`, :guilabel:`Death beneficiary`, or
  :guilabel:`Non-employee.`
- :guilabel:`Income Stream Type`: Select where the employee is receiving their pay from. The typical
  employee is :guilabel:`Salary and wages`, however some other circumstances may require a different
  selection.

> - :guilabel:`Tax Treatment Category`:

Reviewer 3: missing explanation o:

Reviewer: **Tax Treatment Category**: Your STP Phase 2 report will include a 6-character tax treatment code for each employee. The tax treatment code is an abbreviated way of telling us about factors that can influence the amount you withhold from payments to your employees. Your STP solution will automate the reporting of these codes and ensure that the tax treatment code you report is valid. Even though the creation of this code will be automated for you, it is still part of your STP report. It is important for you to understand what it means. For more details, check the ATO website for: [How to report employment and tax information through STP Phase 2](https://www.ato.gov.au/businesses-and-organisations/hiring-and-paying-your-workers/single-touch-payroll/in-detail/single-touch-payroll-phase-2-employer-reporting-guidelines/how-to-report-employment-and-tax-information-through-stp-phase-2)

- :guilabel:`HELP / STSL`: Tick this checkbox if the employee is a :abbr:`STSL (Study Training
  Support Loan)` recipient.
- :guilabel:`Medicare Variation Form`:

Leave loading / workplace giving section
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

- :guilabel:`Eligible for Leave Loading`:
- :guilabel:`Workplace Giving Employee`:

> - :guilabel:`Salary Sacrificed Workplace Giving`:

Reviewer 3: missing explanations! o:

Salary sacrifice section
~~~~~~~~~~~~~~~~~~~~~~~~

- :guilabel:`Salary Sacrifice Superannuation`: Enter the amount per pay period the employee provides
  to the *company*, which then puts that amount into the employee's :abbr:`RESC (Reportable Employer
  Superannuation Contributions)` account, which is an Australian retirement account.
- :guilabel:`Salary Sacrifice Other Benefits`: Enter the amount per pay period the employee puts
  aside towards some other benefit. Refer to the `ATO's web page on Salary sacrificing for employees

>   <https://www.ato.gov.au/individuals-and-families/jobs-and-employment-types/working-as-an-employee/salary-sacrificing-for-employees>`_.

Reviewer 3: ```suggestion
  <https://www.ato.gov.au/individuals-and-families/jobs-and-employment-types/working-as-an-employee/salary-sacrificing-for-employees>`_
```

Author: I didn't even SEE that!

  for more information.

.. note::
   As of Odoo 18, salary sacrificing for other benefits currently does not impact :abbr:`FBT (Fringe

>    Benefits Tax)` reporting.

Reviewer 3: In general we avoid directly reference versions, and when we do, it's only in the version of the documentation we're in.

We can leave this note in the 18 documentation, but it should be removed after and treated as expected behavior!

```suggestion
   Salary sacrificing for other benefits does not affect :abbr:`FBT (Fringe
   Benefits Tax)` reporting.
```

Super accounts tab
------------------

A super account is the term for an Australian retirement savings account. This tab houses the
employee's various retirement accounts, or *super funds*.

Click :guilabel:`add a line`, and a :guilabel:`Create Super Accounts` pop-up window loads. Enter the

> following information in the pop-up window:

Reviewer 3: I believe we need to capitalize "A"? Also we use italics for window popups now (: (I think you knew this but this PR has been sitting for so long haha)

```suggestion
Click :guilabel:`Add a line`. In the *Create Super Accounts* pop-up window, enter the
following information:
```

- :guilabel:`Employee`: Ensure the employee is selected in this field.
- :guilabel:`TFN`: The :abbr:`TFN (Tax Filing Number)` entered in the *Payroll* tab of the employee
  form populates this field, and **cannot** be modified.
- :guilabel:`Member Since`: By default, the current date populates this field. Select the date the
  employee created the super fund.
- :guilabel:`Active`: Enable this option if the account is active (this is the likely scenario).
- :guilabel:`Proportion`: Enter the percentage amount being added to the account. By default, the
  field displays `100`.
- :guilabel:`Super Fund`: Using the drop-down menu, select the specific kind of account the super

>   fund is associated with.

Reviewer 3: optional!

```suggestion
- :guilabel:`Super Fund`: In the drop-down menu, select the type of account associated with the super
  fund.
```

- :guilabel:`Fund ABN`: This field is autopopulated with the selected :guilabel:`Super Fund`
  :abbr:`ABN (Australian Business Number)`, and **cannot** be modified.
- :guilabel:`Member Number`: After selecting a :guilabel:`Super Fund`, this field appears. Enter the
  member number for the employee's super fund account.

.. image:: australia/create-super.png
   :alt: A Create Super Accounts form all filled out.

.. tip::
   Manage all super funds by navigating to :menuselection:`Payroll --> Configuration --> Super
   Funds`, and manage all super accounts by navigating to :menuselection:`Payroll --> Configuration
   --> Super Accounts`.

.. important::

>    Odoo is currently in the process of becoming SuperStream-compliant.

Reviewer 3: earlier in the doc, there was a note claiming that we are already super stream compliant. I'm confused. Are we compliant, or are we in the process of being compliant?

> .. _payroll/au_config:

Reviewer 3: custom anchor style

```suggestion
.. _payroll/payroll_localizations/au_config:
```

:guilabel:`Name` of the rule, and make any edits as needed.

.. important::
   Odoo adds updated rule parameters for the current calendar year. It is **not** recommended to
   edit rule parameters **unless a national or local parameter has changed**, and is different from
   the rule parameters created by Odoo. Check with all local and national regulations *before*
   making any changes to rule parameters.

Run Australian payroll
======================

Before running payroll, the payroll officer must validate employee :doc:`work entries
<../work_entries>` to confirm pay accuracy and catch errors. This includes checking that all time
off is approved and any overtime is appropriate.

Work entries sync based on the employee's :doc:`contract <../contracts>` configuration. Odoo pulls
from the assigned working schedule, attendance records, planning schedule, and approved time off.

Any :ref:`discrepancies or conflicts <payroll/conflicts>` must be resolved, then the work entries
can be :ref:`regenerated <payroll/regenerate-work-entries>`.

Once everything is correct, draft payslips can be :ref:`created individually <payroll/process>` or

> in batches, referred to in the **Payroll** app as :doc:`*Pay Runs* <../pay_runs>`.

Reviewer 3: extra *?

```suggestion
in batches, referred to in the **Payroll** app as :doc:`Pay Runs <../pay_runs>`.
```

Author: This was italicized for emphasis because it's a defintion - but no worries, will remove!

.. important::
   In Australia, :ref:`individual payslips <payroll/process>` (not part of a pay run) are considered
   *out-of-cycle* runs. The same payslip rules apply, but the way individual payslips are submitted

>    to the ATO in the framework of :abbr:`STP (Single Touch Payroll)` is slightly different.

Reviewer 3: ```suggestion
   to the |ATO| in the framework of :abbr:`STP (Single Touch Payroll)` is slightly different.
```

>    As of Odoo 18, adding an out-of-cycle payslip to an existing batch is **not** recommended.

Reviewer 3: ```suggestion
   Adding an out-of-cycle payslip to an existing batch is **not** recommended.
```

.. image:: australia/au-work-entries.png
   :alt: The work entries for a pay run, with some time off entered in the work entries.

.. note::
   To cut down on the payroll officer's time, it is typical to process payslips in batches, either
   by wage type (fixed salary vs hourly), pay schedule (weekly, bi-weekly, monthly, etc.),

>    department (direct cost vs. administration), or any other grouping that best suits the company.

Reviewer 3: attempted to clarify the wording a bit (: optional!

```suggestion
   To reduce payroll processing time, payroll officer typically handle payslips in batches. These batches can be grouped by wage type (e.g., fixed salary or hourly), pay schedule (weekly, bi-weekly, or monthly),
   department (direct cost or administration), or any other structure that suits the company.
```

the amount withheld from payroll taxes is correct, the amount that the employee receives as their
net salary is correct, and the computation of hours worked reflects the employee's actual hours
worked, among others.

When running a payroll batch, check that the period, company, and employees included are correct
*before* starting to analyze or validate the data.

Once the payslips are drafted, review them for accuracy. Check the :guilabel:`Worked Days & Inputs`
tab, and ensure the listed worked time is correct, as well as any other inputs. Add any missing
inputs, such as commissions, tips, reimbursements, that are missing.

Next, check the various totals (gross pay, employee taxes, benefits, employer taxes, net salaries),
then click :guilabel:`Compute Sheet` to update the salary calculations, if there were edits. If
everything is correct, click :guilabel:`Create Draft Entry`.

.. image:: australia/au-check-payslips.png
   :alt: The worked days tab of a payslip.

Accounting check
----------------

The accounting process when running payroll has two components: :ref:`creating journal entries
<payroll/australia/journal>`, and :ref:`registering payments <payroll/australia/register>`.

> .. _payroll/australia/journal:

Reviewer 3: ```suggestion
.. _payroll/payroll_localizations/journal:
```

Journal entry creation
~~~~~~~~~~~~~~~~~~~~~~

After payslips are confirmed and validated, journal entries are posted either individually, or in a
batch. The journal entry is created first as a draft.

.. important::
   It must be decided if journal entries are done individually or in batches *before* running
   payroll.

Four accounts from the Australian :abbr:`CoA (Chart of Accounts)` are included with the payroll
localization:

- `611000 Salaries & Wages`:
- `230100 Employee Payroll Taxes`:
- `230200 Employer Payroll Taxes`:

> - `230000 Salary Payable`:

Reviewer 3: missing explanations!

Reviewer 4: @Felicious 

Extra Compulsory Super %: An additional superannuation percentage that must be contributed on top of the standard super guarantee rate.

Reviewer 4: Also missing Super accounts

  state or industry according to the type of work (e.g., Saturday rate, Sunday rate, overtime rate,
  etc.).

Close Payroll
-------------

If there are no errors, payroll is completed for the pay period.

Reports
=======

The Australian localization contains several reports unique to Australia, which provide information
on superfund contributions, single touch payroll, :abbr:`STP (Single Touch Payroll)`  Finalisation,
and the ability to terminate employees.

STP Finalisation
----------------

The fiscal year in Australia ends on June 30th, but :abbr:`STP (Single Touch Payroll)` finalisation
declarations are due by July 14th. These can be done in :ref:`one single report for all employees
<payroll/stp-total>`, or :ref:`individually <payroll/stp-individual>`.

.. _payroll/stp-total:

EOFY finalisation
~~~~~~~~~~~~~~~~~

To finalize the fiscal year's :abbr:`STP (Single Touch Payroll)`, navigate to
:menuselection:`Payroll app --> Reporting --> STP Finalisation`, and a :guilabel:`Finalise Payroll`
pop-up window loads.

The :guilabel:`Company`, :guilabel:`ABN`, and :guilabel:`Branch Code` are populated from the

> :ref:`company information <payroll/au_gen>`.

Reviewer 3: ```suggestion
:ref:`company information <payroll/payroll_localizations/au_gen>`.
```

Configure the following fields on the form:

> - :guilabel:`BMS ID`:

Reviewer 3: missing definition!

  Taxation Office)`.
- After an employee is :ref:`terminated <payroll/au-terminate>` during the year.

To create an individual finalisation, navigate to :menuselection:`Payroll app --> Reporting --> STP
Finalisation`.

Configure the top-half of the form in the same manner as the :ref:`typical annual finalization
<payroll/stp-total>`, except **do not** tick the checkbox next to the :guilabel:`EOFY Declaration`.
Leaving this unchecked displays a :guilabel:`Deadline Date` field. This is populated with the
current date, by default. Adjust this date if needed.

Next, manually add the employees to be finalised by clicking :guilabel:`Add a line` in the
:guilabel:`Employee Finalisation` tab, and selecting the employee.

.. important::
   If an employee record is finalised partway through the financial year, the ATO will **not**
   pre-fill the information into the employee's tax return until **after** the end of the financial
   year.

.. _payroll/au-terminate:

Terminate employee
------------------

Th terminate an Australian employee, navigate to :menuselection:`Payroll app --> Reporting -->

> Terminate Employee`, and a :guilabel:`Terminate Employee` pop-up window loads.

Reviewer 3: ```suggestion
Terminate Employee`, and a *Terminate Employee* pop-up window loads.
```

- :guilabel:`Employee Type`: Select the *type* of employee.
- :guilabel:`Contract Type`: Determines how the employee is paid and classified, such as
  :guilabel:`Permanent`, :guilabel:`Temporary`, :guilabel:`Apprenticeship`.
- :guilabel:`Pay Category`: Select :guilabel:`Australia: Employee` for this field. This defines when
  the employee is paid, their default working schedule, and the work entry type it applies to.

Schedule section
~~~~~~~~~~~~~~~~

- :guilabel:`Work Entry Source`: Defines how :doc:`work entries <../work_entries>` are generated for
  payroll during the specified pay period. The options are:

  - :guilabel:`Working Schedule`: Based on the employee's assigned :ref:`working schedule
    <employees/schedule>` (e.g., 40 hours per week).
  - :guilabel:`Attendances`: Based on :doc:`approved checked-in hours
    <../../attendances/management>` in the **Attendances** app.
  - :guilabel:`Planning`: Based on :ref:`scheduled shifts <planning/shifts>` in the **Planning**
    app.

- :guilabel:`Working Hours`: Using the drop-down menu, select the default work schedule. This is
  particularly important for employees available to receive overtime pay (typically hourly

>   employees, not salaried). The common selection is :guilabel:`AU Standard 38 hours/week`.

Reviewer 4: remove this as the same is above

Australia
=========

.. |YTD| replace:: :abbr:`YTD (Year to Date)`
.. |ATO| replace:: :abbr:`ATO (Australian Taxation Office)`

The Australian payroll localization covers salary computations for employees, including both
employee and employer payroll taxes. It accounts for all local and national regulations.

Before configuring the Australia localization, refer to the general :doc:`payroll <../../payroll>`
documentation, which includes the basic information for all localizations, as well as all universal
settings and fields.

.. important::
   It is **not** recommended for companies to use the **Payroll** app for the following business
   flows:

   - Income stream types: *Foreign Employment Income*
   - Tax treatment category: *actors & performers*
   - Death benefits reporting
   - Reporting obligations for :abbr:`WPN (Withholding Payer Number )` (instead of :abbr:`ABN
     (Australian Business Number)`)

>    - Allowances subject to a varied rate of withholding (such as cents per kilometer allowance and

Reviewer 4: this point needs to be removed

------------------------------

As per |ATO| requirements, :abbr:`STP (Single Touch Payroll)` submissions for a pay run **must** be
done *on or before* the payday. For this reason, submit :abbr:`STP (Single Touch Payroll)` data to
the |ATO| **before** proceeding with payment. To submit the data, click :guilabel:`Submit to ATO` on
the pay run.

.. tip::
   A warning message appears if important information is missing.

On the :abbr:`STP (Single Touch Payroll)` record for the pay run, an automatically generated
activity for the :abbr:`STP (Single Touch Payroll)` responsible user appears, and a summary of
payslips contained in the pay run, which is able to be audited from this view.

Once the :abbr:`STP (Single Touch Payroll)` record is ready, click :guilabel:`Submit to ATO`, then
read and accept the related terms and conditions.

.. image:: australia/payroll-stp-record.png
   :alt: Example of an STP record.

.. important::
   Odoo has a partnership with a clearing house to process both superannuation payments and data to
   the right funds in one click, via direct debit.

   Odoo is currently in the process of becoming compliant with SuperStream, and an announcement will

>    be made as soon as superannuation contributions can be processed via Odoo's payroll solution.

Reviewer 4: Delete this
